src/lxc/conf.c

```diff
@@ -2342,8 +2343,21 @@ static int lxc_setup_netdev_in_child_namespaces(struct lxc_netdev *netdev)
 				return -1;
 			}
 		}
-		if (netdev->type != LXC_NET_VETH)
+
+		if (netdev->type != LXC_NET_EMPTY)
+			return 0;
+
+		if (netdev->type != LXC_NET_NONE)
 			return 0;
+
+		if (netdev->type != LXC_NET_VETH) {
```

Comment:
You'll never get to the code in this block since you've already ensured that netdev->type == LXC_NET_EMPTY :)  if i'm reading the patch right.